crates/parser/src/parse/libpg_query_node.rs

```diff
@@ -412,7 +412,23 @@ impl<'p> LibpgQueryNodeParser<'p> {
 }
 
 /// list of aliases from https://www.postgresql.org/docs/current/datatype.html
-const ALIASES: [&[&str]; 2] = [&["integer", "int", "int4"], &["real", "float4"]];
+const ALIASES: [&[&str]; 15] = [
+    &["bigint", "int8"],
+    &["bigserial", "serial8"],
+    &["bit varying", "varbit"],
+    &["boolean", "bool"],
+    &["character", "char"],
+    &["character varying", "varchar"],
+    &["double precision", "float8"],
+    &["integer", "int", "int4"],
+    &["numeric", "decimal"],
+    &["real", "float4"],
+    &["smallint", "int2"],
+    &["smallserial", "serial2"],
+    &["serial", "serial4"],
+    &["time with time zone", "timetz"],
```

Comment:
This will not work, because we are comparing token by token, and this text will be split up over multiplen tokens. It requires a larger change.

Reply:
should this be skipped for now?

Reply:
I would suggest to skip, bit if you like to get a bit more involved feel free to implement it